sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -3675,6 +3706,83 @@ int CNPC_Combine::SelectFailSchedule( int failedSchedule, int failedTask, AI_Tas
 	return BaseClass::SelectFailSchedule( failedSchedule, failedTask, taskFailCode );
 }
 
+#ifdef EZ2
+ConVar npc_combine_obstruction_behavior( "npc_combine_obstruction_behavior", "1" );
```

Comment:
Should this default 0 so we can turn it on in skill.cfg?

Reply:
Would it make a particularly important difference?

Reply:
Yes. It's best practice so we can merge the code without actually activating the feature. I wouldn't be opposed to you activating it in skill.cfg right now, though, even though the code hasn't been merged. It sounds like that is no different but it is only because it helps us track the change.